msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/crosscloud/TestCase1420484.java

```diff
@@ -0,0 +1,91 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NON INFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.msal.automationapp.testpass.crosscloud;
+
+import com.microsoft.identity.client.msal.automationapp.AbstractMsalUiTest;
+import com.microsoft.identity.client.ui.automation.browser.BrowserChrome;
+import com.microsoft.identity.internal.testutils.labutils.LabConstants;
+import com.microsoft.identity.internal.testutils.labutils.LabUserQuery;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TestCase1420484 extends AbstractMsalUiTest {
+
+    @Before
+    public void setup() {
+        mActivity = mActivityRule.getActivity();
+        mBrowser = new BrowserChrome();
+    }
+
+    @After
+    public void cleanup() {
+        mBrowser.clear();
+    }
+
+    /**
+     * Tests Acquiring token for Cross cloud Guest account without broker
+     */
+    @Test
+    public void test_1420484() throws Throwable {
+        CrossCloudGuestAccountTests.testAcquireToken(
+                mActivity,
+                mBrowser,
+                LabConstants.GuestHomeAzureEnvironment.AZURE_US_GOV,
+                null);
+
+        mBrowser.clear();
+
+        CrossCloudGuestAccountTests.testAcquireToken(
+                mActivity,
+                mBrowser,
+                LabConstants.GuestHomeAzureEnvironment.AZURE_CHINA_CLOUD,
+                null);
+    }
+
+    @Override
+    public String[] getScopes() {
+        return null;
```

Comment:
Very strange....this test shouldn't pass without any scopes

Reply:
I needed to override it as I am extending the AbstractMsalUiTest class. But this isnt being used in the testcase

Reply:
Well, I think if you are overriding that class then you should use these....or if you feel you don't need these then probably don't override the class...because if these are overriden then the immediate thought to mind is that these are being used and seeing them return null causes confusion

Reply:
I think I will create a separate base class for Guest Account tests. to avoid any immediate thoughts ;)

Reply:
Created a separate base class for guestaccount tests